hdr_html_canvas_element.md

```diff
@@ -18,28 +18,63 @@ There are four main classes of HDR use that inform this proposal. They are:
 
 There exist the following constraints.
 
-* The exact maximum luminance of the output display is not known and not knowable.
+* The exact maximum luminance of the output display is not knowable without user permission.
```

Comment:
This is imprecise, because there are two quantities with different definitions and this text does not say which one is meant. One is the small-area flash luminance and the other is the sustained full-screen peak luminance. See the DisplayHDR conformance test specifications:
https://displayhdr.org/performance-criteria-cts1-1/

Reply:
From the Windows documentation, it seems like this is the small-area value:

> The maximum luminance, in nits, that the display attached to this output is capable of rendering; **this value is likely only valid for a small area of the panel**.

Reply:
I removed this section and moved it to the privacy constraints section (which mentions several fingerprinting vectors, including some of these).